src/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -82,7 +82,7 @@ public GameData loadGame(final ObjectInputStream input, final String savegamePat
         // same engine as was
         // used for this savegame, and if so try to run it
         try {
-          final String newClassPath = TripleAProcessRunner.findOldJar(readVersion, true);
+          final String newClassPath = GameRunner.findOldJar(readVersion, true);
```

Comment:
This on the other hand is the oppositie, TripleAProcessor has a lot of static functions and is closely related to the internals of GameRunner. Consolidating the two allowed more variables to be private between the two classes.